src/design-tokens/sizes.yml

```diff
@@ -1,49 +1,57 @@
 props:
-  edge_small:
+  - name: edge_small
```

Comment:
I just want to ask a naive question. Is the `name` key just an optional organizing principle for Theo YAML files (meaning the consumer output is the same with or without)? I saw this and immediately thought "This must not break anything, right?" Thanks

Reply:
Theo can be an array or an object.

If it is an object, the syntax is more terse but the order of tokens is not preserved.

If it is an array, the order is preserved but you need to use `name` for the variable.

More info: https://github.com/salesforce-ux/theo#spec

Reply:
Ah, I think I saw that in a JSON example, but didn't realize this was the YAML equivalent. Makes sense. Thanks.

Reply:
I missed the leading `-`s, which turned the properties from an object into an array.